specification/containerservice/resource-manager/readme.md

```diff
@@ -59,7 +59,22 @@ input-file:
   - Microsoft.ContainerService/stable/2017-07-01/containerService.json
   - Microsoft.ContainerService/stable/2019-08-01/location.json
   - Microsoft.ContainerService/stable/2019-11-01/managedClusters.json
+
+tag: package-2019-10-27-preview
 ```
+
+### Tag: package-2019-10-27-preview
+
+These settings apply only when `--tag=package-2019-10-27-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-10-27-preview'
+input-file:
+  - Microsoft.ContainerService/preview/2019-10-27-preview/openShiftManagedClusters.json
+  - Microsoft.ContainerService/stable/2017-07-01/containerService.json
```

Comment:
We Should use lastest version from managedClusters.json

This block should be : 
```
- Microsoft.ContainerService/preview/2019-10-27-preview/openShiftManagedClusters.json
- Microsoft.ContainerService/stable/2017-07-01/containerService.json
- Microsoft.ContainerService/stable/2019-08-01/location.json
- Microsoft.ContainerService/stable/2020-01-01/managedClusters.json
```